package.json

```diff
@@ -1,38 +1,24 @@
 {
-  "name": "cesium-webpack-example",
+  "name": "cesiumjs-webpack-tutorial",
   "version": "1.0.0",
-  "description": "The minimal recomended setup for an app using Cesium with Webpack.",
+  "description": "",
   "main": "index.js",
-  "keywords": [
-    "cesium",
-    "webpack",
-    "example"
-  ],
-  "author": {
-    "name": "Cesium GS, Inc.",
-    "url": "https://cesium.com"
-  },
-  "homepage": "https://cesiumjs.org",
-  "license": "Apache-2.0",
-  "dependencies": {
-    "cesium": "^1.63.1"
+  "scripts": {
+    "build": "node_modules/.bin/webpack --config webpack.config.js",
+    "start": "node_modules/.bin/webpack serve --config webpack.config.js --open"
   },
+  "author": "",
+  "license": "ISC",
```

Comment:
We should keep the **"author"** and **"license"** as *"Cesium GS, Inc."* and *"Apache-2.0"*.

so does the **"name"**, **"description"** and **"keywords"**